src/Catty/ViewController/Upload/UploadViewController.swift

```diff
@@ -424,7 +424,17 @@ class UploadViewController: UIViewController, UploadCategoryViewControllerDelega
                                 }
                                 DispatchQueue.main.async(execute: {
                                     if let projectId = projectId {
-                                        project.rename(toProjectName: project.header.programName, andProjectId: projectId, andShowSaveNotification: true)
+                                        project.rename(toProjectName: project.header.programName, andProjectId: projectId, andShowSaveNotification: false)
+
+                                        AlertControllerBuilder.alert(title: kLocalizedProjectUploaded, message: kLocalizedProjectUploadedBody)
+                                            .addDefaultAction(title: kLocalizedSafariShow) {
```

Comment:
If you add ",handler: {" after the kLocalizedSafariShow and move the if inside you could close with "})" and therefore the .addDefaultAction, .build and .showWithController would be aligned and easier to read.

Reply:
This maybe in addition to the changes Michaels change request :D